src/helpers/index.ts

```diff
@@ -19,10 +19,13 @@ export const toTitleCase = (string: string): string => {
 
 export const convertAgencyRecordToSelect =
     (data: AgencyDTO[]): SelectData[] => data.map(choice => {
-      const { label, title } = choice;
+      const { 
+        label, 
+        sys_id // eslint-disable-line camelcase 
+      } = choice;
       return {
         text: label,
-        value: title
+        value: sys_id // eslint-disable-line camelcase

```

Comment:
Changing from `title` to `sys_id` works currently in the DISA sandbox instance on SNOW (with the changes in https://github.com/dod-ccpo/atat-snow/pull/198). However, this might cause another issue when moving to the DSP instance since the `sys_id` will more than likely be different on that instance when the move happens. 

We may want to leave it as `title` to make it a little more robust as the `title` should be consistent across instances. The Platform team will add a bug ticket to change the logic of the business rules to avoid using the `sys_id` and hopefully avoid this bug from happening on a different SNOW instance.